test/smoke_spec.js

```diff
@@ -0,0 +1,73 @@
+var PageObject = require('../page_objects/page_object_model')
+
+describe('smoke test demo', function() {
+var pageObject
+
+beforeEach(function(){
+
+  pageObject = new PageObject()
+
+  browser.waitForAngularEnabled(false);
+
+  browser.get('http://automationpractice.com/index.php');
+
+})
+
+
+      it('shall not smoke', function() {
+  
+      expect(browser.getTitle()).toEqual('My Store');
+
+      pageObject.signinButton.click();
+      
+      pageObject.emailField.sendKeys('abhtestinginternship@gmail.com');
+      pageObject.passwordField.sendKeys('!Qwerty123');
```

Comment:
Can you investigate is there way to keep password out of the code, and to put it as parameter in console run command ?

Reply:
I can try perhaps with environment variables